### PR TITLE
Removed unwanted emoji from the doc

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose
Remove  'A' emoji that was in the Example 3 and fix some alignments.

## Goals
Removed  'A' emoji that was in the Example 3 and fixed the alignments.

## Documentation
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Comment by niveathika** [comment]
@Amstrom the files under docs folder are auto-generated at release time. Update the description on @Extension annotation in the relevant class to make this change persistent,

**Comment by mohanvive** [comment]
@sahandilshan Can you please incorporate the suggestion and send those changes

File: docs/api/2.0.4-SNAPSHOT.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# API Docs - v2.0.4-SNAPSHOT
```
Comment (mohanvive) [inline docs/api/2.0.4-SNAPSHOT.md]: Remove this file

File: mkdocs.yml [hunk 2]
```diff
@@ -34,6 +34,7 @@ pages:
   - Information: index.md
   - API Docs:
       - latest: api/latest.md
+      - 2.0.4-SNAPSHOT: api/2.0.4-SNAPSHOT.md
```
Comment (mohanvive) [inline mkdocs.yml]: Remove this entry.

**Review by mohanvive — CHANGES_REQUESTED** [review]

**Review by mohanvive — APPROVED** [review]

**Pull request merged**